llvm/lib/Target/AArch64/AArch64InstrGISel.td

```diff
@@ -216,7 +216,7 @@ def G_PREFETCH : AArch64GenericInstruction {
 }
 
 // Generic bitwise insert if true.
-def G_BIT : AArch64GenericInstruction {
+def G_BSP : AArch64GenericInstruction {
```

Comment:
And this opcode needs a bit of documentation since it's not a real AArch64 instruction.